docs/configuration.md

```diff
@@ -132,7 +132,65 @@ headers (for example, to set the `Accept:` header to request JSON responses).
 
 [Kubernetes container probes]: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
 
+## Tagging
+
+The NATS role above was tagged as `clustered`, causing fissile to emit
+it as a [StatefulSet].
+
+The second way of causing fissile to do that is to tag a role as
+`indexed`. The main difference between `clustered` and `indexed` roles
+is that fissile creates a public service (of type `LoadBalancer`) for
+the latter, providing a single point of access to the pods for the
+role.
+
+Therefore index roles which require load balancing and need a 0-based,
+incremented index, and cluster otherwise.
+
+Note that both `clustered` and `indexed` roles can take advantage of
+volume claim templates for local storage.
+
+An example of an indexed role is the BLOBSTORE of CF. See the example
+below.
+
+```yaml
+roles:
+- name: blobstore
+  jobs:
+  - name: blobstore
+    release_name: capi
+  processes:
+  - name: blobstore_nginx
+  - name: blobstore_url_signer
+  tags:
+  - indexed                      # Mark this role as indexed => StatefulSet
+  run:
+    scaling:
+      min: 1
+      max: 1
+      # Not HA capable; use external HA storage instead
+    capabilities: []
+    persistent-volumes:          # Local persistent storage, shared across the set
```

Comment:
This is *not* shared across the set; that's `shared-volumes`.
Also, since #341 landed, it should be written as:
```yaml
volumes:
- path: /var/vcap/store
  tag: blobstore-data
  type: persistent # or type: shared, type: host
  size: 50
```

Reply:
Will fix the `shared` thing.
WRT the 341, did we have SCF and UAA PRs fixing the syntax in the respective role manifests ?

Reply:
No, because the old syntax is still supported.

Reply:
Ok. So I don't have to fix up my SCF/UAA work in that regard either.
Reworked the example. Commit 345848333b28aa12daf75028b9ad103163fe782c.